corehq/apps/accounting/models.py

```diff
@@ -2171,13 +2171,14 @@ def get_domain(self):
         return self.subscription.subscriber.domain
 
     @classmethod
-    def autopayable_invoices(cls, date_due=None):
+    def autopayable_invoices(cls, date_due=Ellipsis):
```

Comment:
The conventional way to write `Ellipsis` is `...`
```suggestion
    def autopayable_invoices(cls, date_due=...):
```